### PR TITLE
Correct static site build errors

### PR DESCRIPTION
**Pull request by btylerburton**
Related to:
- https://github.com/GSA/data.gov/issues/4143

### CODE REVIEW

**Comment by jbrown-xentity** [comment]
Any idea why we need `nokogiri@1.13.10-x86_64-linux`, instead of just `nokogiri@1.13.10`? Snyk says the non-linux one is better. I think we can ignore the exception either way, as I believe nokogiri is just used to do internal qa check (and not for the end user), but not sure I understand what's changing and why.

**Comment by nickumia-reisys** [comment]
The commit that they reference might be different, but I think the reason it changed was because of how gem was updated (whether it was run in a container or locally or some other way).  Gem probably chose the OS-specific version for this reason.  This is fully speculation, don't take this as truth haha..

File: Gemfile.lock [hunk 1]
```diff
@@ -62,34 +64,37 @@ GEM
...
     typhoeus (1.4.0)
       ethon (>= 0.9.0)
-    unicode-display_width (2.3.0)
+    unicode-display_width (2.4.0)
     webrick (1.7.0)
     yell (2.2.2)
 
 PLATFORMS
-  ruby
+  x86_64-darwin-21
```
Comment (btylerburton) [inline Gemfile.lock:96]: @jbrown-xentity @nickumia-reisys, re: the nokogiri change, I upgraded bundler to 2.x.x and apparently now the platforms are now much more specific.

Comment (btylerburton) [reply Gemfile.lock:96]: This comes down to building native gems for ARMs and JRuby when available, but I'm not sure of the exact ramifications for our pipeline. Let me do some more digging. Thoughts in the meantime?

Comment (nickumia-reisys) [reply Gemfile.lock:96]: No thoughts, I understand it, but I guess this is just a bad snyk integration more than it is a problem 😕

Comment (btylerburton) [reply Gemfile.lock:96]: after reading the release notes, it looks like a snyk problem as they released the same patch to `nokogiri@1.13.10-x86_64-linux`

https://github.com/sparklemotion/nokogiri/releases/tag/v1.13.10

**Comment by btylerburton** [comment]
@nickumia-reisys @jbrown-xentity added context to snyk ignore, which i'll copy down into other repos

File: Gemfile.lock (hunk 1 above)
Comment (btylerburton) [reply Gemfile.lock:96]: this feels resolved

**Review by jbrown-xentity — APPROVED** [review]

**Pull request merged**